.github/workflows/action-test-smoke.yml

```diff
@@ -22,13 +22,13 @@ jobs:
       - name: Download artifacts
         uses: actions/download-artifact@master
         with:
-          name: containerd-shim-${{ inputs.runtime }}-linux-musl
+          name: containerd-shim-${{ inputs.runtime }}-x86_64-linux-musl
```

Comment:
interesting, I don't see where this arch was being included in the `inputs.runtime` before.  Or is this extra field new?

Reply:
Before this PR, only the release pipeline runs both `x86_64` and `aarch64`. The ci does not have a matrix for different archs. I added that to make the build job on par with the one in the release CI.